index.js

```diff
@@ -3,19 +3,18 @@ const cors = require('cors');
 const { PokemonService } = require('./service');
 const swaggerUI = require('swagger-ui-express');
 const swaggerDoc = require('./swagger.json');
+const { PokemonController } = require('./controller');
+
+const pokemonService = new PokemonService();
+const allPokemons = new PokemonController(pokemonService);
```

Comment:
Name this `pokemonController` 
```suggestion
const allPokemons = new PokemonController(pokemonService);
const pokemonController = new PokemonController(new PokemonService());
```